packages/protocol/engine/notarization/attestation.go

```diff
@@ -62,3 +63,20 @@ func (a Attestation) Bytes() (bytes []byte, err error) {
 func (a *Attestation) FromBytes(bytes []byte) (consumedBytes int, err error) {
 	return serix.DefaultAPI.Decode(context.Background(), bytes, a, serix.WithValidation())
 }
+
+func (a *Attestation) IssuerID() identity.ID {
+	return identity.NewID(a.IssuerPublicKey)
```

Comment:
Seems like this computed value is used often. Maybe we can just create a non-serialized property on object creation to avoid calling this over and over again?

Reply:
This does not seem to be called so often